### PR TITLE
Fix console/nginx test module and its scheduling

### PR DESCRIPTION
**Pull request by pdostal**
- Related tickets: [poo#105786](https://progress.opensuse.org/issues/105786) [poo#105768](https://progress.opensuse.org/issues/105768)
- Previous PR: #14082
- Verification run: [sle-15-SP3-mau-webserver](http://pdostal-server.suse.cz/tests/13174) [Tumbleweed](http://pdostal-server.suse.cz/tests/13172)

### CODE REVIEW

**Comment by punkioudi** [comment]
Pavel, could you also add the VRs? :)

**Comment by pdostal** [comment]
> Pavel, could you also add the VRs? :)

This is in progress.

File: lib/services/nginx.pm [hunk 1]
```diff
@@ -48,7 +48,7 @@ sub config_service {
     assert_script_run "echo '127.0.0.1 vhost' >> /etc/hosts";
     assert_script_run "echo '::1 vhost' >> /etc/hosts";
 
-    common_service_action 'nginx', $service_type, 'reload';
+    common_service_action 'nginx', $service_type, 'restart';
```
Comment (DimStar77) [inline lib/services/nginx.pm:51]: I doubt this will magiclly make apache stop

**Review by okurz — APPROVED** [review]

**Comment by pdostal** [comment]
This is now from my opinion ready to be reviewed and/or merged.

**Review by DimStar77 — APPROVED** [review]

**Review by punkioudi — APPROVED** [review]
lgtm!

**Pull request merged**